src/ewatercycle/base/forcing.py

```
@@ -416,7 +424,7 @@ def _build_recipe(
         shape: Path,
         dataset: Dataset | str | dict,
         variables: tuple[str, ...] = (),
-        **model_specific_options,
+        **model_specific_options,  # noqa: ARG003
```

A bit annoying to keep having to repeat this noqa comment. Perhaps it's better to just disable it completely in this module.

---

I like that its explicit for those lines. Having a module wide ignore, could hide errors in the rest of the module.